projects/prosoft-components-demo/src/app/common/demo-ps-form-service.ts

```diff
@@ -0,0 +1,16 @@
+import { BasePsFormService, IPsFormError, IPsFormErrorData } from '@prosoft/components/form-base';
+import { Observable, of } from 'rxjs';
+
+export class DemoPsFormsService extends BasePsFormService {
+  public getLabel(formControl: any): Observable<string> {
+    return formControl.psLabel ? of(formControl.psLabel) : null;
+  }
+  protected mapDataToError(errorData: IPsFormErrorData[]): Observable<IPsFormError[]> {
```

Comment:
mapDataToError$?

Reply:
Would be a breaking change and I thought about changing the return type to Observable<IPsFormError> | IPsFormError, because our ne translation doesn't need observable and it would be less runtime overhead without observable.